spec/features/hierarchical_response_display_logic_spec.rb

```diff
@@ -0,0 +1,386 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "hierarhcical response form display logic", js: true do
+  def visit_new_hierarchical_response_page
```

Comment:
Can helper methods go at the bottom please?

Reply:
Will move into context